src/scss/_vars_mixins.scss

```diff
@@ -1,25 +1,20 @@
 $o-header-is-silent: true !default;
-$o-header-primary-height: 45px;
-$o-header-primary-background: #333333;
-$o-header-secondary-background: oColorsGetPaletteColor('grey-tint5');
+$o-header-bottom-height: 45px;
+// width of the left column in mobile view of the menu
+// i.e. the headers of nav sections
+$o-header-mobile-width: 5;
 
 @mixin oHeaderNavItem {
-	color: oColorsGetPaletteColor('white');
-	text-decoration: none;
 	display: block;
-	line-height: $o-header-primary-height;
 	padding: 0 4px;
-	font-size: 14px;
-	background: none;
 	border: 0;
+	background: none;
+	color: inherit;
+	font-size: 14px;
```

Comment:
Cool, happy to stick to `px`, was mostly just curious. The discussion and SO article linked to is good. Just had a quick test as well on iOS to make sure the accessibility options for large text don't effect websites and they don't, which is good.